src/molecules/Navbar/NavbarContainer.tsx

```diff
@@ -14,21 +17,24 @@ export default styled.div<NavbarContainerProps>`
     align-items: center;
     justify-content: center;
     z-index: ${theme.zIndex.navbar};
-
-    ${fixed &&
-      css`
-        position: fixed;
-        top: 0;
-        left: 0;
-        right: 0;
+      ${position &&
```

Comment:
maybe this should be `position === 'absolute'` so we don't support anything random?

Reply:
I thought it wasn't necessary, since the `position` prop only takes `fixed` or `absolute`, so I was under the impression that passing any other value would raise an error like
```
Type '"random"' is not assignable to type '"fixed" | "absolute" | undefined'
```
But yeah, I'll add it just in case.